provider/resources.go

```diff
@@ -2510,44 +2510,25 @@ func Provider() tfbridge.ProviderInfo {
 			},
 			"google_identity_platform_tenant": {Tok: gcpResource(gcpIdentityPlatform, "Tenant")},
 
-			// Diagflow
-			"google_dialogflow_agent":          {Tok: gcpResource(gcpDiagflow, "Agent")},
```

Comment:
why are these no longer necessary?

Reply:
Because we have automatic token mapping:

https://github.com/pulumi/pulumi-gcp/blob/3740714b0e32bdf0cff499580a1611f99edbb86e/provider/resources.go#L3890-L3894

The change is unrelated to setting `DeleteBeforeReplace: true`.

Reply:
makes sense, thanks